### PR TITLE
changes to SessionPage tutor and student sections

### PR DESCRIPTION
**Pull request by SOLEIGH**

### CODE REVIEW

File: amplify/backend/function/Practice/Practice-cloudformation-template.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
+{
```
Comment (albonkey) [inline amplify/backend/function/Practice/Practice-cloudformation-template.json]: Practice function should be removed.

File: src/features/sessions/sessionInfoSlice.js [hunk 2]
```diff
@@ -34,8 +34,9 @@ export const getSession = (sessionID) => async (dispatch) => {
 
         const {data} = await API.get('tutorhubAPI', `/sessions/${sessionID}`);
 
-        dispatch(sessionInfoSuccess(data.Items[0]));
+        dispatch(getSessionSuccess([...data.Items]));
```
Comment (albonkey) [inline src/features/sessions/sessionInfoSlice.js]: This should not be array. Just one Session.

File: src/features/sessions/sessionInfoSlice.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ import {API} from 'aws-amplify';
 export const sessionInfoSlice = createSlice({
     name: 'session',
     initialState: {
-        session: {}
+        session: []
```
Comment (albonkey) [inline src/features/sessions/sessionInfoSlice.js]: Change to object.

**Review by albonkey — CHANGES_REQUESTED** [review]

**Review by SOLEIGH — COMMENTED** [review]
Removed practice function
Left array in SessionInfoSlice because SessionDetails is querying entire Session object (details, student, tutor)

**Pull request merged**